### ISSUE TITLE
How to make Discord desktop app trust mitmproxy certificate?

### ISSUE THREAD

**Issue by EmmieShayz** [Roachbones/discordless#14]
Whenever I try to use mitmproxy with the Discord desktop app, it displays the error `Client TLS handshake failed. The client does not trust the proxy's certificate for discord.com (OpenSSL Error([('SSL routines', '', 'sslv3 alert certificate unknown')]))`

In the documentation it says to "add the certificate in Chrome", but I'm unsure what that means.

**Comment by Roachbones** [comment]
By "add the certificate in Chrome", I meant that I opened my Chrome browser settings, found the "Manage certificates" settings, and imported the certificate there. Once I did that, the Discord desktop app trusted the certificate. Are you on Linux too?

**Comment by BilonGamer** [comment]
That is what I did, although it didn't seem to have an effect. I'm on Windows 10.

I worked around this by downgrading to an earlier version of mitmproxy, which seemed to work. Although now this issue seems to have gone away on newer versions of mitmproxy too, however there is a different problem going on with them. Should I create a new issue for it?

**Comment by recursivetree** [comment]
Yes please create a new issue if you have another problem. In the meantime, I'll close this one if the original issue seems to have gone away, but feel free to reopen it if it comes back.
